content/articles/2018-11-parsnip-0-0-1.Rmarkdown

```diff
@@ -212,6 +218,8 @@ glmnet(x = x_mat, y = car_train$mpg, alpha = 0, lambda = 0.1)
 
 `alpha = 0` tells `glmnet` to only use an L2 penalty (as opposed to L1  and L2). 
 
+AH: the "possible syntax here make it unclear to me whether or not this actually works"
```

Comment:
Do you mean that about the `glmnet` code above or the `keras` code below?

Reply:
the `keras` code below

Reply:
I'll make a note about that